### PR TITLE
Add Parsing of TargetRectangle to ResizeWebProcessor

### PR DESCRIPTION
**Pull request by stefannikolei**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp.Web/pulls) open
- [x] I have verified that I am following matches the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description
Adds the ability to specify the targetrectangle in the commands. As asked in #302

### CODE REVIEW

**Comment by stefannikolei** [comment]
Attention this targets the v3 branch

File: src/ImageSharp.Web/Processors/ResizeWebProcessor.cs [hunk 1]
```diff
@@ -134,7 +157,8 @@ public class ResizeWebProcessor : IImageWebProcessor
             Mode = GetMode(commands, parser, culture),
             Compand = GetCompandMode(commands, parser, culture),
             Sampler = GetSampler(commands),
-            PadColor = parser.ParseValue<Color>(commands.GetValueOrDefault(Color), culture)
+            PadColor = parser.ParseValue<Color>(commands.GetValueOrDefault(Color), culture),
+            TargetRectangle = targetRectangle
```
Comment (JimBobSquarePants) [inline src/ImageSharp.Web/Processors/ResizeWebProcessor.cs:161]: I think we can just use `new(0 ,0, size.Width, size.Height)`. We don't do source/target rectangles in this library.

Comment (JimBobSquarePants) [reply src/ImageSharp.Web/Processors/ResizeWebProcessor.cs:151]: Ah no... 

I meant don't add these sommnds at all and use the already parsed width/height.

Comment (stefannikolei) [reply src/ImageSharp.Web/Processors/ResizeWebProcessor.cs:151]: But then the rectangle will always be created when a width and height is set. Would that be ok. When we add them we are probably more flexible, or doesn't it make sense to use different once?

**Review by JimBobSquarePants — APPROVED** [review]
Looks great. Thanks! 👍

**Pull request merged**